Makefile

```diff
@@ -1,9 +1,9 @@
-include common.mk
+all: hello
 
-all:
+include $(azul_home)/common.mk
```

Comment:
Chicken-egg. common.mk actually tests for presence of azul_home. Must use relative path.